lib/friendly_id/sequentially_slugged.rb

```diff
@@ -26,7 +26,7 @@ def initialize(scope, slug, slug_column, sequence_separator)
       end
 
       def next_slug
-        slug + sequence_separator + next_sequence_number.to_s
+        [slug, next_sequence_number].compact.join(sequence_separator)
```

Comment:
I think it's better to fix this problem this way: https://github.com/norman/friendly_id/pull/720.